l10ninsp/slave.py

```diff
@@ -123,6 +132,21 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    curPath = os.path.dirname(os.path.abspath(__file__))
+    vendorLocalPath = os.path.join(curPath, '..', '..')
+    cljsPath = os.path.join(vendorLocalPath, 'compare-locales.js')
+    # nodejs ./bin/compare-dirs.js --data=json ~/elmo/stage/workdir/a
+    process = subprocess.Popen(
+      ["nodejs",
```

Comment:
OK.  However, I think we should do this the other way round: specify `node` here as this is the official name of the binary as per the project maintainers.  For Debian/Ubuntu we can recommend https://packages.debian.org/sid/nodejs-legacy